packages/flutter/lib/src/painting/image_provider.dart

```diff
@@ -332,6 +332,74 @@ abstract class ImageProvider<T> {
     return ImageStream();
   }
 
+  /// Checks if this [ImageProvider] has a matching key in the
```

Comment:
Based on this comment I expected a true/false responds. Instead I get the location in the cache.

Reply:
Updated the comment.